frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -66,6 +69,14 @@ private void processIndentationElement(PsiElement element, AnnotationHolder hold
 
     IndentationParameters indentationParameters = findIndentationParameters(element);
 
+    // Potentially unrelated - it's responsible to change default TAB key behavior
+    // (insert real TAB or insert some number of spaces) depending on detected file indentation.
+    // In case of space insertion indent width (size) is also set
```

Comment:
Very unlikely case, but I think a indent width >1 is also possible in case of tab

Reply:
(but nobody is gonna do that in real life anyway)

Reply:
Yes, but it seems to be not supported by IntelliJ  :/